### PR TITLE
Resolves #239: add company level endpoint to get cloud tenant billing.

### LINKED ISSUE #239: Add company endpoint method for getting cloud tenant billing

### PR DESCRIPTION
**Pull request by bsnyder788**

### CODE REVIEW

File: src/sdk/model/company/__json__/currency-type.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export type Currency = 'AUD' |
```
Comment (cfsnyder) [inline src/sdk/model/company/__json__/currency-type.ts]: we already have this enum in common/billing

File: src/sdk/model/company/__json__/cloud-tenant-bill-json.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { CurrencyCodeJson } from './currency-code-json';
+
+export interface CloudTenantBillJson {
+  total: number;
+  year: number;
+  month: number;
+  currency_code: CurrencyCodeJson;
```
Comment (cfsnyder) [inline src/sdk/model/company/__json__/cloud-tenant-bill-json.ts]: why does the api not just place the enum value in this property directly vs. having another object as a layer of indirection?

Comment (bsnyder788) [reply src/sdk/model/company/__json__/cloud-tenant-bill-json.ts]: I just used the auto generated ones. I can change it.

Comment (bsnyder788) [reply src/sdk/model/company/__json__/cloud-tenant-bill-json.ts]: or do you mean java side @cfsnyder ?

File: src/sdk/model/company/__json__/currency-type.ts (hunk 1 above)
Comment (bsnyder788) [reply src/sdk/model/company/__json__/currency-type.ts]: 👍

File: src/sdk/model/company/__json__/cloud-tenant-bill-json.ts (hunk 2 above)
Comment (cfsnyder) [reply src/sdk/model/company/__json__/cloud-tenant-bill-json.ts]: yeah i meant java side

**Comment by bsnyder788** [comment]
@cfsnyder  updated to match the new model (core side PR is open)

**Pull request merged**